biome.json

```diff
@@ -7,7 +7,8 @@
       ".output/",
       ".vinxi/",
       "routetree.gen.ts",
-      "packages/icons-react/src"
+      "packages/icons-react/src",
+      "apps/docs/docgen.ts"
```

Comment:
Bør filen gitignores i stedet for å sjekkes inn? Den bygges jo compile time. Ellers havner fort i en situasjon hvor vi ender opp med en git diff på denne filen, når man egentlig jobber med andre ting.

Reply:
Absolutt!